src/doc.d

```diff
@@ -379,6 +379,7 @@ RPAREN= )
 BACKTICK= `
 DOLLAR= $
 DEPRECATED= $0
+NOTHING =
```

Comment:
Per http://wiki.dlang.org/Contributing_to_dlang.org, you can define an actual space like this:

```
SPACE = $(SPACE) $(SPACE)
```

Note the actual space in between the two macro expansions (which will expand to nothing).